api/types/client.go

```diff
@@ -138,6 +138,21 @@ func (h *HijackedResponse) CloseWrite() error {
 	return nil
 }
 
+// ImageSaveOptions holds the information
+// necessary to save a set of images.
+type ImageSaveOptions struct {
```

Comment:
Do these need to be added to the API schema? @dnephin

Reply:
It looks like these are only used in the client interface, not in the HTTP API, so they do not need to be adding to the swagger schema.